Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -183,7 +210,7 @@
      */
     protected AbstractForestUpdateExecutor<?> updateExecutor;
 
-    protected PointStoreDouble pointStore;
+    protected IPointStore<?> pointStore;
```

Comment:
Can we remove this instance variable now? Our goal is that all state should be owned by the update coordinator or the component list.

Reply:
Done.